test/unit/gateways/borgun_test.rb

```diff
@@ -56,6 +56,23 @@ def test_authorize_and_capture
     assert_success capture
   end
 
+  def test_authorize_airline_data
+    # itinerary data abbreviated for brevity
```

Comment:
Maybe it's just me, but I don't worry too much about brevity in tests, if that was the only thing holding you back from adding more! That being said, this much is probably fine, too. What do you think?

Reply:
If I were actually testing that each item was making it into the XML, I think it'd make sense to include all of them. However, since the API seems lmore of ike it doesn't follow a strict ordering (today in history!!) I didn't feel it was necessary to test the inclusions specifically. 

This might be an argument for eliminating even more of the included data! But I didn't want it to be totally unrecognizable either. 🤷‍♀

Reply:
Yep, I think it's fine here. Just didn't know if brevity was the _only_ reason it was being shortened. Huzzah for any-order XML!